src/main/java/org/sonar/plugins/jacoco/ReportPathsProvider.java

```diff
@@ -39,18 +43,27 @@
   }
 
   Collection<Path> getPaths() {
-    Set<Path> reportPaths = Stream.of(context.config().getStringArray(REPORT_PATHS_PROPERTY_KEY))
-      .map(this::toAbsolutePath)
-      .collect(Collectors.toSet());
+    Set<Path> reportPaths = new HashSet<>();
+
+    for (String pathName : context.config().getStringArray(REPORT_PATHS_PROPERTY_KEY)) {
+      Path path = toAbsolutePath(pathName);
+      if (reportExists(path)) {
+        reportPaths.add(path);
+      } else {
+        LOG.warn("Coverage report doesn't exist: '{}'", path);
+      }
+    }
 
     if (!reportPaths.isEmpty()) {
       return reportPaths;
-    }
+    } else {
+      LOG.info("Can not find JaCoCo reports from property 'sonar.coverage.jacoco.xmlReportPaths', using default locations...");
```

Comment:
why the tree dots? Maybe we should display WHAT are the default locations instead of letting people guess.